aptos-move/block-executor/src/executor.rs

```diff
@@ -848,71 +980,43 @@ where
                         0,
                         "Sequential execution must materialize deltas"
                     );
-                    // Apply the writes.
-                    for (key, write_op) in output
-                        .resource_write_set()
-                        .into_iter()
-                        .map(|(k, (v, _))| (k, v))
-                        .chain(output.aggregator_v1_write_set().into_iter())
-                        .chain(output.module_write_set().into_iter())
-                    {
-                        data_map.write(key, write_op);
-                    }
```

Comment:
is this now in incorporate_materialized_txn_output?

Reply:
This is inside execute_transactions_sequential. I separated resource_write_set (which has MoveTypeLayout) from module and aggregator v1 write set.